tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -4424,8 +4424,13 @@ Status ConvertSquare(OpConverterParams* params) {
   const auto& inputs = params->inputs;
   const auto& node_def = params->node_def;
   TF_RETURN_IF_ERROR(CheckInputsWeights(*params, {{"x", false}}));
+#if IS_TRT_VERSION_GE(6, 0, 1, 0)
```

Comment:
We have a plan to provide a way to get the supported data types for all op converters. The code here is fine for this reason.